proto/kvrpcpb.proto

```diff
@@ -25,9 +25,9 @@ message KeyError {
 }
 
 enum CommandPri {
-    Normal = 0;
-    Low = -1;
-    High = 1;
+    Normal = 0; // Normal must the default value
```

Comment:
do we have many cases to use low priority? if not, the negative is not a problem.